lib/nimble.phx.gen.template/addons/ex_coveralls.ex

```diff
@@ -0,0 +1,69 @@
+defmodule Nimble.Phx.Gen.Template.Addons.ExCoveralls do
+  use Nimble.Phx.Gen.Template.Addon
+
+  @versions %{
+    excoveralls: "~> 0.12.2"
+  }
+
+  @impl true
+  def do_apply(%Project{} = project, _opts) do
+    project
+    |> copy_files
```

Comment:
Let's always add parenthesis when piping even there is no argument 👍 

```suggestion
    |> copy_files()
```

Reply:
is there a way to set it as invalid with mix format @rosle? so we could avoid this issue in the future

Reply:
@rosle I think Credo handles this issue, I will open a PR to adding Credo to the template itself, so we could fix this issue.